### ISSUE TITLE
Will we soon have the new version of Genesis Plus GX 1.7.5 (Linux) available ?!?

### ISSUE THREAD

**Issue by marcelosofth** [ekeeke/Genesis-Plus-GX#326]
Friend "ekeeke" First I would like to thank Genesis Plus GX today for being the best Mega Drive and Master System emulator for Retroarch, I am very happy and grateful for that and I would like to thank you for all the development, I realized since the latest version 1.7.4 was released a long time ago and 1.7.5 has been in the oven for a long time, is there the possibility of this long-awaited release?

**Comment by ekeeke** [comment]
I have stopped bumping the versions since a long time because emulator development has kinda slowed down for me and is more continuous development when I got free time to add some stuff I like or when someone reports an issue, without any real planned 'milestones' or scheduled 'official' releases.

There is no linux version distributed here so I assume you are speaking about the libretro port in Retroarch, which is normally always built from the latest source in libretro repository so the version you get from automatic core downloader should always be up-to-date with the latest commited changes.

So basically it's not 1.7.4 anymore since a long time but it doesn't really matter.
